db-scripts/src/main/resources/gen-cgds-test-schema.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+CGDS_SCHEMA_FILE=$(find . -type f -name "*cgds.sql" | grep -v target)
```

Comment:
please add comments to this file explaining why it is needed and when it could be removed again (e.g. when h2db adds support for partial indexes)

Reply:
@pieterlukasse how does this sound?


```
# The purpose of this script is to create a test version of the database schema to load in:
#     `./persistence/persistence-mybatis/src/test/resources/testContextDatabase.xml`
#
# The only difference between the production version of `./db-scripts/src/main/resources/cgds.sql` and the
# version loaded for testing is the dropping of the partial index on `mutation_event`.`TUMOR_SEQ_ALLELE`.
#
# This is done to accommodate the H2 MySQL driver since the latest version (1.4.197) does not yet
# support partial indexing.
#
# When the H2 MySQL driver finally does support partial indexing, this script can be removed and the production
# version can be used for testing again.
#
# author: ochoaa
# last update: 2019/02/13
```

Reply:
cc @sheridancbio ☝️

Reply:
@angelicaochoa excellent. Thanks.